[DOC-562] Update build the docs page with instructions for Ubuntu and a specific hugo version. (#25121)
yugabyte/yugabyte-db: docs/content/preview/contribute/docs/ubuntu.md

<!--
+++
private=true
+++
-->

> * **Python** (version 3.10 or earlier): You need `python` to be available somewhere in your shell path.

Reviewer: ```suggestion
* **Python** (v3.10 or earlier): You need `python` to be available somewhere in your shell path.
```

>     Recent versions of Ubuntu default to `python` and point to Python 3. If not, you can install new version of Python, and use [pyenv](https://github.com/pyenv/pyenv) to manage multiple Python versions. Ensure you're using Python 3.10 or earlier.

Reviewer: ```suggestion
    Recent versions of Ubuntu default to `python` and point to Python 3. If not, you can install a new version of Python, and use [pyenv](https://github.com/pyenv/pyenv) to manage multiple Python versions. Ensure you're using Python 3.10 or earlier.
```

> * **Hugo**: Install `hugo_extended_0.131.0` version from the [official Hugo releases](https://github.com/gohugoio/hugo/releases) as per your Linux machine architecture e.g `amd`, `arm`, and `64bit`. Make sure the `hugo` binary is available in the shell path.

Reviewer: ```suggestion
* **Hugo**: Install the `hugo_extended_0.125.6` version from the [official Hugo releases](https://github.com/gohugoio/hugo/releases) for your Linux machine architecture (for example, `amd`, `arm`, `64bit`). Make sure the `hugo` binary is available in the shell path.
```

* **Go**: Install latest Go from the [official Go website](https://golang.org/dl/).

> * **Git client**: Ubuntu includes Git by default, but to ensure you have the latest version (`sudo apt-get install git`).

Reviewer: ```suggestion
* **Git client**: Ubuntu includes Git by default, but ensure you have the latest version (`sudo apt-get install git`).
```
